kornia/contrib/models/rt_detr/post_processor.py

```diff
@@ -24,32 +28,42 @@ def forward(self, data: dict[str, Tensor], meta: dict[str, Any]) -> list[Tensor]
             Processed detections. For each image, the detections have shape (D, 6), where D is the number of detections
             in that image, 6 represent (class_id, confidence_score, x, y, w, h).
         """
-        logits = data["logits"]
-        boxes = data["boxes"]
-        original_sizes = meta["original_size"]
+        # NOTE: consider using kornia BoundingBox
+        # NOTE: consider passing the parameters as separate arguments instead of a dict
```

Comment:
i would love to understand why started using plain dicts in forward calls 😄

Reply:
unfortunately is the trend across research code